scripts/add-utxo.py

```diff
@@ -231,7 +231,8 @@ def main():
     else:
         quit(parser, 'Invalid syntax')
     if options.validate or options.vonly:
-        if not validate_utxo_data(utxo_data, segwit=True):
+        sw = False if jm_single().config.get("POLICY", "segwit") == "false" else True
```

Comment:
to avoid anything confusing from happening it would probably be better to have the same check as https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/816078b437d9a0b6d31be4d2b827582602717abc/jmclient/jmclient/wallet.py#L571

maybe there should be a global utility function/variable for this

Reply:
Oh thanks for the heads up, didn't notice you'd done it that way round: I always wanted to have the logic the other way round so a user can only use non-segwit if they explicitly choose it with "false". I think I said it that way in the docs somewhere. So I'd rather edit your version.